.azure-pipelines/docker-sonic-vs/Dockerfile

```diff
@@ -5,6 +5,11 @@ ARG need_dbg
 
 COPY ["debs", "/debs"]
 
+# Remove existing packages first before installing the new/current packages. This is to overcome limitations with
+# Docker's diff detection mechanism, where only the file size and the modification timestamp (which will remain the
+# same, even though contents have changed) are checked between the previous and current layer.
+RUN dpkg --purge libswsscommon python3-swsscommon sonic-db-cli libsaimetadata libsairedis libsaivs syncd-vs swss sonic-eventd
```

Comment:
What about swss-dbg from line 22?

Reply:
Debug packages aren't installed in the docker-sonic-vs container that's built as part of sonic-buildimage, so that doesn't need to be removed.